Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,8 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+const minArr = tasks.map((x) => x.duration);
+const hoursArr = minArr.map((y) => y / 60).filter((y) => y >= 2);
+const perHour = hoursArr.map((x) => x * 10);
+const sum = perHour.reduce((x) => x + x);
```

Comment:
This use of reduce is a bit incorrect.
I would suggest reading
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce

`const sum = perHour.reduce((acc, x) => acc + x, 0);`
The zero is optional, but generally good form, because there are lots of times it is important to have an initial value.